Framework/API/src/FileFinder.cpp

```diff
@@ -741,8 +787,24 @@ const API::Result<std::string> FileFinderImpl::getPath(const std::vector<IArchiv
     }
   }
 
+  // Search data cache
+  string errors;
+  std::filesystem::path cachePathToSearch(
+      Mantid::Kernel::ConfigService::Instance().getString("datacachesearch.directory"));
+  if (std::filesystem::exists(cachePathToSearch)) {
+
+    API::Result<std::string> cacheFilePath = getDataCachePath(cachePathToSearch.string(), filenames, exts);
+
+    if (cacheFilePath) {
+      return cacheFilePath;
+    }
+    errors += cacheFilePath.errors();
+
+  } else {
+    errors += "Could not find data cache directory: " + cachePathToSearch.string();
+  }
```

Comment:
Will this always happen when not on IDAaaS? If so will it be annoying / confusing?

Reply:
No, if the file is then found on the archive then the errors string is never displayed. Otherwise, if the file is neither at the data cache nor at the archive the errors string contains both the data cache and archive errors, and will be displayed in the tooltip of the `Load` function.

Reply:
Ah, so the errors only appear if the file isn't found anywhere

Reply:
Yes, exactly, but even then it doesn't show as an error message, it goes into the tooltip